test/functional/search_controller_test.rb

```diff
@@ -361,21 +361,6 @@ def no_results
     end
   end
 
-  test "should send analytics headers for citizen proposition" do
-    result = {
-      "title" => "title",
-      "link" => "/slug",
-      "highlight" => "",
-      "format" => "publication"
-    }
-    stub_results([result], "bob", [], [], total: 1)
-    get :index, {q: "bob"}
-    assert_equal "search",  @response.headers["X-Slimmer-Section"]
-    assert_equal "search",  @response.headers["X-Slimmer-Format"]
-    assert_equal "citizen", @response.headers["X-Slimmer-Proposition"]
```

Comment:
Should it be just this line that gets removed?

From what I can see the test is poorly named and was actually testing all analytics headers, the others are still being used.